show/main.py

```diff
@@ -1074,7 +1082,8 @@ def version(verbose):
     click.echo("Serial Number: {}".format(chassis_info['serial']))
     click.echo("Model Number: {}".format(chassis_info['model']))
     click.echo("Hardware Revision: {}".format(chassis_info['revision']))
-    click.echo("Uptime: {}".format(sys_uptime.stdout.read().strip()))
+    click.echo("Uptime: {}".format(get_uptime()))
```

Comment:
May be its ok to add a new '`date`' field but wouldn't suggest changing the existing format of `Uptime `as it may break backward compatibility.

Reply:
@prsunny having the time and uptime seems like not a correct approach. It just provide info which is not relevant to the tag.
If we will run the sonic-mgmt test to ensure that the change does not create an issue will it be OK?

Reply:
@prsunny I run the sonic-mgmt test on this image, and it passes it successfully.

Reply:
@EdenGri , show version's uptime output is from the command `uptime `which has the current time. Let's not change it. Suggest you add the new 'date' field and retain the uptime field as is.

Reply:
@prsunny Done.